app/models/report.rb

```diff
@@ -482,4 +482,128 @@ def self.report_trending_search(report)
       }
     end
   end
+
+  def self.report_moderator_activity(report)
+    report.data = []
+    mod_data = {}
+    mod_ids = []
+
+    User.real.where(moderator: true).select(:id, :username).as_json.each do |mod|
```

Comment:
Slightly more efficient 

```ruby
mod_data = {}

User.real.where(moderator: true).pluck(:id, :username).each do |u| 
  mod_data[u[0]] = { user_id: u[0], username: u[1] }
end

mod_ids = mod_data.keys
```

Reply:
I currently do not know the whole context and it's just a quick lookup - but wouldn't it be more efficient to replace `each` with `find_each`?